backend/src/server.js

```diff
@@ -15,5 +16,5 @@ app.use(express.json());
 app.get('/',(req,res) => {
     res.status(200).json("hello");
 })
-
-app.listen(port, ()=> console.log(`App server listening on port ${port}!`));
\ No newline at end of file
+mongoose. connect('mongodb://localhost:27017/stins', {useNewUrlParser: true})
```

Comment:
Are we using mongoDB locally? Or are we using Atlas?

Reply:
This was for local, the next story is connecting with MongoDB Atlas

Reply:
sweet